test/puppetlabs/puppetdb/pql/parser_test.clj

```diff
@@ -37,6 +37,25 @@
      "nodes"
      [:extract "a" "b" "c"]]
 
+    "nodes [a, <b>, c] {}"
```

Comment:
Just so I'm clear, we're testing that we can parse this thing because the parser allows it, but this would fail if it ever made it to the query engine, right?